builder/vsphere/common/step_remove_cdrom.go

```diff
@@ -17,33 +18,79 @@ import (
 type RemoveCDRomConfig struct {
 	// Remove CD-ROM devices from template. Defaults to `false`.
 	RemoveCdrom bool `mapstructure:"remove_cdrom"`
+	// Reset CD-ROM device without any attached media. Defaults to `false`.
+	// `remove_cdrom` must be set to `true` when using this option.
```

Comment:
Given the description and its reliance on remove_cdrom being true I feel like reset might not be want we want. Should this be eject_cdrom? 
```
// Eject CD-ROM remove the attached media from the CD-ROM but does not remove it from the template. Defaults to `false`.
// `eject_cdrom` can not be set when `remove_cdrom` is `true`. 
```